### ISSUE TITLE
Error: unable to build kubernetes objects from release manifest: error validating ""

### ISSUE THREAD

**Issue by Wolfsrudel** [zalando/postgres-operator#1252]
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?** registry.opensource.zalan.do/acid/postgres-operator:v1.5.0
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** Bare Metal Openshift
- **Are you running Postgres Operator in production?** no
- **Type of issue?** Bug report and/or question

Trying to install the operator in my Openshift environment (4.5.0-0.okd-2020-10-29-044318) I am getting the following error message:

```
git clone https://github.com/zalando/postgres-operator.git
cd postgres-operator.git
helm install postgres-operator ./charts/postgres-operator -f ./charts/postgres-operator/values-crd.yaml -n "${_namespace}" --create-namespace

coalesce.go:199: warning: destination for cluster_labels is a table. Ignoring non-table value application:spilo
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(OperatorConfiguration.configuration.aws_or_gcp): unknown field "enable_ebs_gp3_migration" in do.zalan.acid.v1.OperatorConfiguration.configuration.aws_or_gcp, ValidationError(OperatorConfiguration.configuration): unknown field "enable_pgversion_env_var" in do.zalan.acid.v1.OperatorConfiguration.configuration]

```
Greeping through `charts/postgres-operator/crds/operatorconfigurations.yaml` shows me no match for `enable_pgversion_env_var`.

**Comment by Wolfsrudel** [comment]
My fault. I have to use configTarget: "ConfigMap". rtfm.

**Comment by FxKu** [comment]
Thanks for raising. There were indeed two issues with the helm chart. Fixed them with #1253
